src/tests.rs

```diff
@@ -59,6 +60,22 @@ fn parse_file() {
 }
 
 #[test]
+fn read_write_file() {
+    let mut path = Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf();
```

Comment:
I usually do `#[cfg(test)] extern crate tempdir;` in `lib.rs`.

Unwrapping the result of `TempDir::new` in a test is fine: the test will fail if a temporary directory could not be created.